bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/internal/loader/DefaultScriptFileWatcher.java

```diff
@@ -37,19 +38,19 @@ public class DefaultScriptFileWatcher extends AbstractScriptFileWatcher {
     private static final String FILE_DIRECTORY = "automation" + File.separator + "jsr223";
 
     @Activate
-    public DefaultScriptFileWatcher(final @Reference ScriptEngineManager manager,
-            final @Reference ReadyService readyService, final @Reference StartLevelService startLevelService) {
-        super(manager, readyService, startLevelService, FILE_DIRECTORY);
+    public DefaultScriptFileWatcher(
+            final @Reference(target = WatchService.CONFIG_WATCHER_FILTER) WatchService watchService,
+            final @Reference ScriptEngineManager manager, final @Reference ReadyService readyService,
+            final @Reference StartLevelService startLevelService) {
+        super(watchService, manager, readyService, startLevelService, FILE_DIRECTORY, true);
     }
 
     @Activate
-    @Override
     public void activate() {
         super.activate();
     }
 
     @Deactivate
-    @Override
```

Comment:
Why remove `@Override` if it still overrides the parent method?

Reply:
I have removed the methods, the `activate`/`deactivate` methods are called automatically.